### ISSUE TITLE
Colculating FID Score

### ISSUE THREAD

**Issue by RohollahHS** [luping-liu/PNDM#15]
Great work. Congratulations.

Are the cifar10 stats computed here https://drive.google.com/drive/folders/1RvjSE2AZSa7VrB3jh9ACO-ylVs8Q91cs (using pytorch-fid) statistics from cifar10 train set or test set?

I downloaded cifar10 and computed both train and test statistics, naturally mu and sigma for all of them are close, but none of my computed train stats and tests stats matched your fid stats.

Thanks
